x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -425,13 +425,16 @@ export const RULES_TABLE_MAX_PAGE_SIZE = 100;
 export const RULES_TABLE_PAGE_SIZE_OPTIONS = [5, 10, 20, 50, RULES_TABLE_MAX_PAGE_SIZE];
 
 /**
- * A local storage key we use to store the state of the feature tour UI for the Rule Management page.
+ * Local storage keys we use to store the state of our new features tours we currently show in the app.
  *
- * NOTE: As soon as we want to show a new tour for features in the current Kibana version,
- * we will need to update this constant with the corresponding version.
+ * NOTE: As soon as we want to show tours for new features in the upcoming release,
+ * we will need to update these constants with the corresponding version.
  */
-export const RULES_MANAGEMENT_FEATURE_TOUR_STORAGE_KEY =
-  'securitySolution.rulesManagementPage.newFeaturesTour.v8.2';
+export const NEW_FEATURES_TOUR_STORAGE_KEYS = {
+  RULE_MANAGEMENT_PAGE: 'securitySolution.rulesManagementPage.newFeaturesTour.v8.2',
+  RULE_CREATION_PAGE_DEFINE_STEP:
+    'securitySolution.ruleCreationPage.defineStep.newFeaturesTour.v8.4',
+};
```

Comment:
Added the storage key here because we need to disable this tour in e2e tests, so the key ideally needs to be easily discoverable.